modules/cloudwatch/locals.tf

```diff
@@ -0,0 +1,4 @@
+locals {
+  # How longs (days) to retain CloudWatch logs
```

Comment:
Do we need this comment? I thought maybe not as days is already mentioned and it's in cloudwatch module

Reply:
I see you removed this comment now so will mark it as resolved.